event_pubsub/producers/blockchain_event_producer.py

```diff
@@ -266,7 +266,7 @@ def _push_events_to_repository(self, events):
 
     def _get_base_contract_path(self):
         return os.path.abspath(
-            os.path.join(os.path.dirname(__file__), '..', '..', 'node_modules', 'singularitynet-platform-contracts'))
+            os.path.join(os.path.dirname(__file__), '..', '..', 'node_modules', 'singularitynet-stake-contracts'))
```

Comment:
@prashantramangupta will it not break any existing platform contract changes?

Reply:
This is specifically for Token Stake contract